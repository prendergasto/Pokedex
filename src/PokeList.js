import React, { Component } from 'react';
import PokeItem from './Poke-item.js';
import pokemon from './data.js'

export default class PokeList extends Component {
    render() {
        return(
            <div id="displayContainer">
                <ol>
                    {this.props.data.map(pokemon => <PokeItem data={pokemon} />)}
                </ol>
            </div>
        )
    }
}


// data={pokemon}