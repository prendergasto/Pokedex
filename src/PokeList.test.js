import React from 'react';
import PokeList from './PokeList';
import renderer from "react-test-renderer";
test('renders PokeItem.js correctly', () => {
  const tree = renderer
  .create(<PokeList data={[]}/>)
  .toJSON();
  expect(tree).toMatchSnapshot();
});

// test passing