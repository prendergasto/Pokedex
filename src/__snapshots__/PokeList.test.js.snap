// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PokeItem.js correctly 1`] = `
<div>
  <ul
    id="displayContainer"
  />
</div>
`;
